### DISCUSSION TITLE
Can we run application developed on UNO platform on Windows 10 IoT Core

### DISCUSSION THREAD

**Discussion by Srikanth-aakuthota** [unoplatform/uno#13531] [Q&A] [answered]
Despite my extensive search across various websites, I'm still unable to find a definitive answer regarding whether an application built on the UNO platform can be executed on Windows 10 IoT Core. According to Microsoft's official documentation, the only applications compatible with Windows 10 IoT Core are UWP apps.

Adding to my confusion is the fact that, to the best of my knowledge, UNO is diligently working on a framework closely aligned with UWP. 

Please see my post in stackoverflow here  -- https://stackoverflow.com/questions/77038896/can-we-run-application-developed-on-uno-platform-on-windows-10-iot-core

**Comment by jeromelaban** [comment]
Thanks for the question, @Srikanth-aakuthota. 

Uno Platform is all about implementing UWP/WinUI on non windows platforms and have the original UWP/WinUI implementations run on Windows.

Are original UWP applications causing issues for your scenario? An what is the scenario that makes Uno compelling for that environment?

**Reply by Srikanth-aakuthota** [reply]
Thank you for the answer @jeromelaban,

We're exploring the possibility of transitioning our Winform application, developed on the .NET Compact Framework, to a Windows application compatible with Windows 10 IoT Core. As the current OS selected for the product is Windows 10 IoT Core. During my research, I discovered that the sole type of Windows application capable of running on Windows 10 IoT Core is the UWP app.

Now, the question arises: Is it feasible for us to opt for the UNO platform as the foundation for constructing our Windows application? that can run now on Windows 10 IoT core?

While I comprehend the UNO platform's primary purpose is to run the applications on non-windows platforms, however it seems we can leverage the current UNO platform offering.  For instance, if we decide to create a mobile application for our product down the line,  we could seamlessly migrate to support Mobile? 

Currently, I've generated a project using the template below and managed to execute a UWP app on Windows 10 IoT Core. It appears that the template provided is essentially a basic UWP application, correct?

![image](https://github.com/unoplatform/uno/assets/6263180/fec217f6-60e4-4777-bd49-eb4d424f2234)

**Reply by jeromelaban** [reply, marked as answer]
Thanks for the details. 

As you envision going to mobile targets, you can convert your code to UWP first to have your application running on Windows 10 IoT Once done, you can migrate the solution to use Uno heads so that your code can run on other platforms as well. We've got some documentation [available here](https://platform.uno/docs/articles/howto-migrate-existing-code.html).
